lumen/readme.md

```diff
@@ -0,0 +1,17 @@
+Run a Lumen site on Caddy
+---------
+
+It is simply neccesary to redirect all requests to index.php
+
+    api.mysite.ie 
+    
+    #rewrite all requests to index.php
+    rewrite / {
+     r (.*)
+     to /index.php
+    }
+    
+    fastcgi / 127.0.0.1:9876 php
+    root "C:\path\tolumen\public"
```

Comment:
I don't really see the point in duplicating the Caddyfile in the readme. It's okay to have a short readme, and it's less to maintain for me and others. So just take this out and I'll merge it in! Thanks.